velox/exec/tests/SpillTest.cpp

```diff
@@ -493,3 +493,43 @@ TEST_F(SpillTest, spillPartitionSet) {
     }
   }
 }
+
+TEST_F(SpillTest, spillPartitionSpilt) {
+  for (int iter = 0; iter < 5; ++iter) {
+    int numBatches = 100;
+    std::vector<RowVectorPtr> batches;
+    batches.reserve(numBatches);
+
+    const int numRowsPerBatch = 100;
+    setupSpillState(
+        iter % 2 ? 1 : 1'000'000'000, 1, numBatches, numRowsPerBatch);
```

Comment:
Is 1'000'000'000 a file size? Consider adding constants like kGB for readability.